### ISSUE TITLE
Default setting doesn't work

### ISSUE THREAD

**Issue by nemesiscompletion** [lllyasviel/stable-diffusion-webui-forge#1129]
With new Forge I installed today, I noticed that default setting doesn't work
I know to change defaults setting, go to Settings tab -> Defaults -> View Changes -> Apply -> Reload UI
However, the settings never saves or shows the last setting

This is what I used for my settings
![20240815_015950](https://github.com/user-attachments/assets/ea2352bd-fa61-4187-9c53-b34d117441d3)

But it keeps shows this one when I restart or reload web UI
![20240815_020245](https://github.com/user-attachments/assets/3ed9b5fa-bcc2-4a3d-8939-c068542275db)

This is the version for my Forge
![20240815_020407](https://github.com/user-attachments/assets/773678dc-961b-47da-aaeb-12570fdc3e62)

Is this some ongoing issue that's going to be fixed on next patch or is it just me having this problem?
If it's the latter, is there way to solve this?
Thank you for your time

**Comment by danilomaiaweb** [comment]
I also waiting for this solution :(

**Comment by R3CoCo** [comment]
I'm getting different numbers in dimensions, but I have the same problem. Even editing in ui-config or config is doing nothing. 

Also I can't get Hires. fix to stay turn on, after restart it is back to off, someone knows where I can change that? Also Schedule type is always back to "Automatic", this is annoying. 

Before the update I was able to set up everything once with notepad and It was ready to go every restart. Good that I have saved version of Forge before the update but it is lacking some features.

**Comment by dandybobel** [comment]
Same for me.

**Comment by hopperelec** [comment]
Having the same issue. If I click "View changes", it correctly shows me what _should_ be the defaults, showing that if I "Apply" then those defaults would be changed to what is actually being used as the defaults. Keep in mind, if looking at the screenshot, that this is what it looks like if I reload the UI then immediately, without making any changes, go to click "View changes".
![image](https://github.com/user-attachments/assets/38934811-3827-4f0d-baac-c6f959a6c964)

**Comment by coghazard** [comment]
Same issue.

**Comment by testonedev** [comment]
My guess is, this bug (?) appeared as a result of introducing the UI profile system for SD, XL, Flux and All... It could be also an incomplete feature, which introduced the profiles, but no separate save functions, yet.

**Comment by R3CoCo** [comment]
> There are two easy **workarounds** for that atm.
> 
>     * You can manually edit the preset values for sd/xl/flux/all in `modules_forge/main_entry.py`, under the function `on_preset_change`.
> 
>     * You can disable the presets system entirely by commenting out some lines in `modules_forge/main_entry.py`.
> 
> 
> For the second option, look for those lines and comment them by placing a # at the beginning of the line:
> 
> ```python
> # example of a comment
> # Those two lines load the preset
>     ui_forge_preset.change(on_preset_change, inputs=[ui_forge_preset], outputs=output_targets, queue=False, show_progress=False)
>     Context.root_block.load(on_preset_change, inputs=None, outputs=output_targets, queue=False, show_progress=False)
> 
> # And this one adds the buttons to choose the preset
>     ui_forge_preset = gr.Radio(label="UI", value=lambda: shared.opts.forge_preset, choices=['sd', 'xl', 'flux', 'all'])
> ```
> 
> By commenting those 3 lines you should now be able to edit the settings through `ui-config.json` and/or `config.json`, and likely through the "apply defaults" option too.
> 
> **Just don't blame me if something breaks because of that**, but if it does you can always revert the changes.
> 
> You might get merge conflicts during updates because of that, or something might just break, so I recommend keeping an eye out for commits that modify this file.

Thank you for this solution, I will try that later

**Comment by GalaxyTimeMachine** [comment]
This still seems to be a problem. The defaults work for some settings, but not others. It doesn't reset the sampler, scheduler, CFG or any of the extensions that were enabled. It's a pain to have to manually go through everything every time you start Forge.

**Comment by Mark-Tomlinson** [comment]
Agreed. Still a problem on version: f2.0.1v1.10.1-previous-636-gb835f24a, checkpoint: 52cfce60d7.

I'd think that a simple solution would be to save the values for the views. But, for now, I just leave mine set to all.

**Comment by scriptcloud** [comment]
No need to manually disable in python, for now just set the preset to 'all' and your default settings would be there.
<img width="230" alt="Image" src="https://github.com/user-attachments/assets/342729d9-6c4a-4828-bfa7-d797403103d8" />
Though I think it being applied to other presets would be the expected behavior. Hope this gets implemented soon.

**Comment by Gebsfrom404** [comment]
Everything above seed kept but extensions still getting reset every time, they do not even show up in defaults/view changes. 
Also prompt text area responsivness just dies after first generation.
[Loom](https://www.loom.com/share/e63947242f9e4d1b873d7d845059aaa9)

**Comment by phil2sat** [comment]
So the bug is open around a year, when fix!?

i know how to set it to ALL but either remove the choice completeley or make it work as it should.

i guess a better way would to load the model from different subfolders,
like sdxl/sd1.5/flux and unlock options depending on folder.

**Comment by AquaSZS** [comment]
Wow, such a basic feature is still broken. It seems Forge is basically an abandonware, and people have moved on to other UI.
